Motion/README.md

```diff
@@ -18,15 +18,25 @@ All the source codes for demos are in the [demo](app/src/main/java/com/example/a
 package.
 
 - [Layout > Dissolve](app/src/main/java/com/example/android/motion/demo/dissolve)
```

Comment:
Let's make all of these headings (h3?) instead of bullets. The indentation looks a bit weird with the images.